usr/share/rear/layout/save/GNU/Linux/230_filesystem_layout.sh

```diff
@@ -231,6 +231,13 @@ fi
                 echo -n " uuid=$uuid label=$label"
                 ;;
             (btrfs)
+                # FIXME: Support for multi-disk BTRFS should be implemented sooner or later
+                # because it is the default layout for multi-disk Fedora Workstation installations.
+                # See: https://github.com/rear/rear/issues/2028
+                if ! grep -q "Total devices 1" <(btrfs filesystem show "$mountpoint"); then
+                    Error "Mounpoint $mountpoint points to a BTRFS filesystem spanning multiple disk devices which is not supported. See: https://github.com/rear/rear/issues/2028"
```

Comment:
```suggestion
                    Error "Mounpoint $mountpoint points to a BTRFS filesystem spanning multiple disk devices which is not yet supported. See: https://github.com/rear/rear/issues/2028"
```

Reply:
Addressed in cc151f3f41455f291356495729aca0a5125b8f13.